internal/server/kong/ws/config/compat/compatibility_table_test.go

```diff
@@ -1148,6 +1149,53 @@ func TestDisableChangeTracking(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "[zipkin] change is emitted with non-default value for" +
+				" http_response_header_for_traceid",
```

Comment:
```suggestion
			name: "[zipkin] change is not emitted with default value for" +
				" http_response_header_for_traceid",
```

Reply:
The default value for `http_response_header_for_traceid` is `null`. The test "`[zipkin] change is emitted with non-default value for http_response_header_for_traceid`" asserts that the upgrade warning is emitted when `http_response_header_for_traceid`, set to a non-default value, is dropped from the config for a incompatible data plane version.

Reply:
If I update this test by removing the [expected change](https://github.com/Kong/koko/blob/71ff2bb014f148ca0cfded61041448701c8612a4/internal/server/kong/ws/config/compat/compatibility_table.go#L856-L876), we get an error:
```
        	Error:      	Not equal: 
        	            	expected: config.TrackedChanges{ChangeDetails:[]config.ChangeDetail(nil)}
        	            	actual  : config.TrackedChanges{ChangeDetails:[]config.ChangeDetail{config.ChangeDetail{ID:"P136", Resources:[]config.ResourceInfo{config.ResourceInfo{Type:"plugin", ID:"759c0d3a-bc3d-4ccc-8d4d-f92de95c1f1a"}}}}}
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,3 +1,13 @@
        	            	 (config.TrackedChanges) {
        	            	- ChangeDetails: ([]config.ChangeDetail) <nil>
        	            	+ ChangeDetails: ([]config.ChangeDetail) (len=1) {
        	            	+  (config.ChangeDetail) {
        	            	+   ID: (config.ChangeID) (len=4) "P136",
        	            	+   Resources: ([]config.ResourceInfo) (len=1) {
        	            	+    (config.ResourceInfo) {
        	            	+     Type: (string) (len=6) "plugin",
        	            	+     ID: (string) (len=36) "759c0d3a-bc3d-4ccc-8d4d-f92de95c1f1a"
        	            	+    }
        	            	+   }
        	            	+  }
        	            	+ }
        	            	 }
Test:       	TestDisableChangeTracking/[zipkin]_change_is_emitted_with_non-default_value_for_http_response_header_for_traceid
```

Reply:
My bad, yes you are right there; please ignore the suggestion. You do need to add a test though to make sure it is emitted.

Reply:
This PR already adds `http_response_header_for_traceid: null` to the [existing default value test](https://github.com/Kong/koko/blob/71ff2bb014f148ca0cfded61041448701c8612a4/internal/server/kong/ws/config/compat/compatibility_table_test.go#L914-L951). This asserts that a change message is not emitted when `http_response_header_for_traceid` set to the default value is dropped from a config for an incompatible data plane. 

Are you requesting that I add a test that asserts that a change message is not emitted when the configuration for a data plane `>= 3.1.0` has `http_response_header_for_traceid` created + set to the default value (due to the request coming from a control plane `< 3.1.0`)?

Reply:
I saw the addition, but missed what it was doing; sorry for the confusion.